tests/index.test.ts

```diff
@@ -0,0 +1,204 @@
+/*
+ * Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import 'mocha';
+import * as setupCloudSDK from '../src/index';
+import { expect } from 'chai';
+import * as io from '@actions/io';
+import { TestToolCache } from '../src/test-util';
+
+const { KEY, B64_KEY, PROJECT_ID, RUNNER_OS } = process.env;
+const [toolDir, tempDir] = TestToolCache.override();
+
+describe('#setupCloudSDK', function() {
+  beforeEach(async function() {
+    await io.rmRF(toolDir);
+    await io.rmRF(tempDir);
+  });
+
+  afterEach(async function() {
+    try {
+      await io.rmRF(toolDir);
+      await io.rmRF(tempDir);
+    } catch (err) {
+      console.error('Error occurred during test cleanup: ' + err);
+    }
+  });
+
+  let version = '349.0.0';
+  before(async () => {
+    if (!KEY || !B64_KEY || !PROJECT_ID) {
+      throw Error('Env Vars not found!');
+    }
+    version = await setupCloudSDK.getLatestGcloudSDKVersion();
+  });
+
+  it('returns false if not installed', function() {
+    const installed = setupCloudSDK.isInstalled();
+
+    expect(installed).eql(false);
+  });
+
+  it('returns false if version is not installed', function() {
+    const installed = setupCloudSDK.isInstalled(version);
```

Comment:
To test this should we install TEST_SDK_VERSIONS[-1] and check isInstalled returns false for TEST_SDK_VERSIONS[-2] ?